src/gromacs/nblib/forcecalculator.cpp

```diff
@@ -312,7 +312,7 @@ gmx::PaddedHostVector<gmx::RVec> ForceCalculator::compute(const bool printTiming
 
     // Todo manage this at a higher level
     nbnxn_atomdata_t*                nbat = nbv->nbat.get();
-    gmx::PaddedHostVector<gmx::RVec> verletForces(nbat->numAtoms(), gmx::RVec(0, 0, 0));
+    gmx::PaddedHostVector<gmx::RVec> verletForces(system_.topology().numAtoms(), gmx::RVec(0, 0, 0));
```

Comment:
In that case I think we can also get rid of `nbnxn_atomdata_t*                nbat = nbv->nbat.get();` no?

Reply:
Yes, we should. Oversight on my part.